### PR TITLE
Attribute Error checked for ConditionSet

### PR DESCRIPTION
**Pull request by darkcoderrises**
Fixes https://github.com/sympy/sympy/issues/10537

### CODE REVIEW

**Comment by smichr** [comment]
Could you add a test for a few of the types that cause problems (like And, Interval and ConditionSet) so we can see what the new results look like?

**Comment by darkcoderrises** [comment]
Is this okay?

File: sympy/printing/pretty/tests/test_pretty.py [hunk 1]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
     assert pretty(ConditionSet(x, Eq(sin(x), 0), S.Reals)) == ascii_str
     assert upretty(ConditionSet(x, Eq(sin(x), 0), S.Reals)) == ucode_str
 
+    assert pretty(ConditionSet(x, Contains(x, S.Reals, evaluate=False), FiniteSet(1))) == \
+        '{x | x in {1} and (Contains(x, (-oo, oo)))}'
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: parentheses don't appear to be needed here

File: sympy/printing/pretty/tests/test_pretty.py [hunk 2]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
...
+    assert upretty(ConditionSet(x, Contains(x, S.Reals, evaluate=False), FiniteSet(1))) == u('{x | x ∊ {1} ∧ (x ∈ ℝ)}')
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: nor here

File: sympy/printing/pretty/tests/test_pretty.py [hunk 3]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
...
+
+    assert pretty(ConditionSet(x, And(x > 1, x < -1), FiniteSet(1, 2, 3))) ==\
+        '{x | x in {1, 2, 3} and (And(x > 1, x < -1))}'
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: nor here

File: sympy/printing/pretty/tests/test_pretty.py [hunk 4]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
...
+    assert upretty(ConditionSet(x, And(x > 1, x < -1), FiniteSet(1, 2, 3))) == \
+        u('{x | x ∊ {1, 2, 3} ∧ (x > 1 ∧ x < -1)}')
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: here, I think they are needed

File: sympy/printing/pretty/tests/test_pretty.py [hunk 5]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
...
+
+    assert pretty(ConditionSet(x, Or(x > 1, x < -1), FiniteSet(1, 2))) == '{x | x in {1, 2} and (Or(x > 1, x < -1))}'
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: not here

Comment (darkcoderrises) [inline sympy/printing/pretty/tests/test_pretty.py]: Could you suggest a method to remove them?

Comment (darkcoderrises) [inline sympy/printing/pretty/tests/test_pretty.py]: Could you suggest a method to fix this?

**Comment by darkcoderrises** [comment]
I fixed that issue, what should be my next step?

File: sympy/printing/pretty/tests/test_pretty.py [hunk 6]
```diff
@@ -3199,6 +3199,18 @@ def test_pretty_ConditionSet():
...
+    assert upretty(ConditionSet(x, Or(x > 1, x < -1), FiniteSet(1, 2))) == u('{x | x ∊ {1, 2} ∧ (x > 1 ∨ x < -1)}')
```
Comment (smichr) [inline sympy/printing/pretty/tests/test_pretty.py]: but here, yes

A test for Interval should be added, too. 

But we should maybe stop and think about this. The second arg to ConditionSet is supposed to be a condition for x and the third argument the interval from which x is selected. None of the examples you have added make sense in light of this. (And all of them, it seems, should have been evaluated, e.g. `ConditionSet(x, Or(x > 1, x < -1), FiniteSet(1,2))` should just give `FiniteSet(2)`. That is a separate issue, however.)

Let's go back and see if it is the instantiation that should have raised an error for bad args -- in which case the printer would not need to be modified: we would just raise an error at instantiation.

Comment (darkcoderrises) [inline sympy/printing/pretty/tests/test_pretty.py]: Consider the case `ConditionSet(x, And(x%4, x%5), S.Integers)`. I think cases like this requires the printer to be updated.

**Comment by smichr** [comment]
OK.

**Pull request merged**